crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -264,9 +269,11 @@ impl From<&Account> for AccountInterface {
         let mut auth = Vec::new();
         components.iter().for_each(|interface| {
             match interface {
-                // RpoFalcon512 and RpoFalcon512Acl use the same RpoFalcon512 auth scheme
+                // RpoFalcon512, RpoFalcon512Acl, and RpoFalconMultisig use the same RpoFalcon512
+                // auth scheme
                 AccountComponentInterface::AuthRpoFalcon512(storage_index)
-                | AccountComponentInterface::AuthRpoFalcon512Acl(storage_index) => {
+                | AccountComponentInterface::AuthRpoFalcon512Acl(storage_index)
+                | AccountComponentInterface::AuthRpoFalconMultisig(storage_index) => {
```

Comment:
This won't give the right result here because for multisig, the storage slot currently actually points to the config slot - which is not the public key. But we can fix this as a part of https://github.com/0xMiden/miden-base/pull/1759.